product_restricted_type/models/product.py

```diff
@@ -8,7 +8,7 @@ class ProductCategory(models.Model):
     _inherit = "product.category"
 
     restricted_product_type = fields.Selection(
-        string="Restricted Product Type",
+        string="Restricted Product Type ",
```

Comment:
I think pre-commit is failing due to string has the same name as the transformed `restricted_product_type` to String. Please remove the string entry